src/language.py

```diff
@@ -27,12 +27,12 @@ def lang_code(self):
 
 # Currently we are only supporting languages and not regions
 class Language(object):
-  JAPANESE = _Language('日本語', 'ja', 'JP')
-  ENGLISH = _Language('English', 'en', 'US')
-  SPANISH = _Language('Español', 'es', 'ES')
-  FRENCH = _Language('Français', 'fr', 'FR')
+  ja = _Language('日本語', 'ja', 'JP')
```

Comment:
I had used uppercase here as an enum value. Is this not really an enum?

Reply:
It is an enum and probably should be in caps for that reason assuming that’s python convention.

However the reason I changed it was to allow the JSON config to be used for #686 and the two digit language code (in lowercase) will make that easier.

I could make it uppercase here and keep the JSON lowercase and just uppercase it when I read it convert it to a language object. WDYT?

Reply:
SGTM